src/views/layouts/GuestLayout/__tests__/component.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import GuestLayout from '../component'
+
+describe('GuestLayout', () => {
+  it('matches snapshot', () => {
+    const Children = () => <>Test</>
+    const component = shallow(
+      <GuestLayout>
+        <Children />
```

Comment:
why just not to pass `'Test'` directly?

Reply:
I check in EB, for layout components used the same pattern